src/codecoverage/FileInfoService.java

```diff
@@ -85,6 +84,24 @@ public FileInfo getFileInfoByFile(File file){
 				//fileInfo.getReportOfTestedFunction();
 				else
 				{
+					File fk = new File("/Users/ten24user/git/codecoverage/htmlOutput.html");
+					   
+			        BufferedWriter bw = new BufferedWriter(new FileWriter(fk,true));
+			       // bw.write("\n\nTest file not found. Hence percentage of coverage is zero.");
+			      //  bw.write("\n\n<----------------------------------------------------------------------------------------->");
+			        bw.write("<tr>");
+			        
+			        bw.write("<td>"+fileInfo.file.getName()+"</td>");
+			        bw.write("<td>"+Boolean.toString(fileInfo.testFile.exists())+"</td>");
+			        bw.write("<td>"+"0"+"</td>");
+			        bw.write("<td>"+"0"+"</td>");
+			        bw.write("<td>"+"0"+"</td>");
+			        
+			        bw.write("<td>"+"0"+"</td>");
+			       
+			        bw.write("</tr>");
+			        bw.write("</table>");
```

Comment:
so this is in the service but probably should not be within the for loop because we only need one table tag. It might make sense to create the table in a service method after all the file information is collected so that way all html code is in one easy to read/write place.